### DISCUSSION TITLE
Live leaderboard scores

### DISCUSSION THREAD

**Discussion by TheBrunoRM** [ppy/osu#16337] [Ideas] [answered]
It would be cool to be able to see the leaderboard scores as they are being played.
For instance, if I had scores on a map, then I would be able to play against my live score, just as if I was on a multi game.
This could also be implemented for the global ranking leaderboard scores.

**Comment by peppy** [comment, marked as answer]
This is what we call "ghost" mode in general. It was already planned for stable but lazer came along as I was working on it.

See https://github.com/ppy/osu/discussions/13119

**Reply by TheBrunoRM** [reply]
I'm looking forward to seeing it in the game!
